packages/patternfly-4/react-core/src/components/Wizard/WizardNavItem.tsx

```diff
@@ -2,33 +2,37 @@ import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/patternfly/components/Wizard/wizard.css';
 
-interface WizardNavItemProps {
+export interface WizardNavItemProps {
+  /** Can nest a WizardNav component for substeps */
   children?: React.ReactNode;
-  label?: string;
-  current?: boolean;
-  disabled?: boolean;
-  onNavItemClick(step: number): any;
+  /** The text to display in the nav item */
+  text?: string;
+  /** Whether the nav item is the currently active item */
+  isCurrent?: boolean;
+  /** Whether the nav item is disabled */
+  isDisabled?: boolean;
+  /** Callback for when the nav item is clicked */
+  onNavItemClick?(step: number): any;
+  /** The step passed into the onNavItemClick callback */
   step: number;
-  hasChildren?: boolean;
 }
 
 export const WizardNavItem: React.FunctionComponent<WizardNavItemProps> = ({
   children = null,
-  label = '',
-  current = false,
-  disabled = false,
+  text = '',
+  isCurrent = false,
+  isDisabled = false,
   step,
-  onNavItemClick,
-  hasChildren = false
+  onNavItemClick
 }: WizardNavItemProps) => (
     <li className={css(styles.wizardNavItem)}>
       <a
-        aria-current={current && !hasChildren ? 'page' : 'false'}
-        onClick={() => onNavItemClick(step)}
-        className={css(styles.wizardNavLink, current && 'pf-m-current', disabled && 'pf-m-disabled')}
-        aria-disabled={disabled ? 'true' : 'false'}
-        tabIndex={disabled ? -1 : undefined}>
-        {label}
+        aria-current={isCurrent && !children ? 'page' : 'false'}
```

Comment:
'false' is that suppose to be a string?

Reply:
Joachim's on PTO, so I'll respond. They're equivalent actually, React will transform the boolean `false` to the string `"false"`. I think it's slightly cleaner as bools, so I'll change it. Have to push to update snapshots anyway :)